### PR TITLE
Integrate academy

### PR DESCRIPTION
**Pull request by winescout**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -312,6 +317,8 @@ DEPENDENCIES
   invoker
   jbuilder (~> 2.0)
   jquery-rails
+  jquery-turbolinks
```
Comment (orangewolf) [inline Gemfile.lock]: @winescout Did we mean to add turbolinks back? I know it can play havoc with Angular

**Comment by orangewolf** [comment]
+∞

File: Gemfile.lock (hunk 1 above)
Comment (StrangeTim) [inline Gemfile.lock]: Steve was using it.  I will use jquery foo to make it unecessary. it's not
a big change

Comment (orangewolf) [inline Gemfile.lock]: =-)

**Pull request merged**